pkg/tracing/tracing.go

```diff
@@ -17,8 +18,16 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+func ShutdownExporter(ctx context.Context, exp *otlptrace.Exporter) {
+	ctx, cancel := context.WithTimeout(ctx, time.Second)
```

Comment:
Allow the caller to control how long to wait on this operation. Don't decide your own timeout here.

Reply:
Same as above